gems/pending/openstack/amqp/openstack_ceilometer_event_monitor.rb

```diff
@@ -0,0 +1,84 @@
+require 'openstack/openstack_event_monitor'
+require 'openstack/amqp/openstack_amqp_event'
+require 'openstack/amqp/openstack_ceilometer_event_converter'
+
+class OpenstackCeilometerEventMonitor < OpenstackEventMonitor
+  def self.available?(options = {})
+    begin
+      options[:ems].connect(:service => "Metering")
+      return true
+    rescue => ex
+      $log.debug("Skipping Openstack Ceilometer events. Availability check failed with #{ex}.") if $log
+    end
+    false
+  end
+
+  def self.plugin_priority
+    1
+  end
+
+  def initialize(options = {})
+    @options = options
+    @ems = options[:ems]
+  end
+
+  def start
+    @since          = nil
+    @monitor_events = true
+  end
+
+  def stop
+    @monitor_events = false
+  end
+
+  def provider_connection
+    @provider_connection ||= @ems.connect(:service => "Metering")
+  end
+
+  def each_batch
+    while @monitor_events
+      begin
+        $log.info("Quering Openstack Ceilometer for events newer than #{latest_event_timestamp}...") if $log
+        query_options = [{
```

Comment:
Would it be sufficient add black/white list of event summary names (instance.* etc) to settings (vmdb.yml) or it is neccessary create a mapping ampq queues to data from ceilometer? Or add it in ongoing PR?